frontend/src/i18/english.ts

```diff
@@ -71,8 +71,8 @@ export const english = {
     'Created on': 'Created on',
     Name: 'Name',
     Value: 'Value',
-    'Value ascending': 'Value, lowest first',
-    'Value descending': 'Value, highest first',
+    'Value ascending': 'Lowest {{type}} first',
+    'Value descending': 'Highest {{type}} first',
```

Comment:
The titles get a little clipped due to lack of space (a horizontal scrollbar does appear though): would only "Lowest/Highest {{type}}" suffice?
![image](https://user-images.githubusercontent.com/80758782/171800819-4e2ff614-fb2c-47d3-98ab-27a80ccf47cb.png)

Reply:
How about anchoring the menu from the right side?

That keeps the menu visible, and allows room for the more complete description for the ordering.